macros/LearnSidebar.ejs

```diff
@@ -1724,7 +1787,33 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
-
+  <li><a href="<%=baseURL%>Tools_and_testing/GitHub"><%=text['Git_and_GitHub']%></a></li>
```

Comment:
I tested it and it works! Please remove this first list element though:

```suggestion
```

Reply:
Oops; I left that in by mistake! Removed now.